wgsl/index.bs

```diff
@@ -1403,6 +1403,9 @@ The [=built-in value=] names are:
 * <a for="built-in values" lt=num_workgroups>`'num_workgroups'`</a>
 * <a for="built-in values" lt=subgroup_invocation_id>`'subgroup_invocation_id'`</a>
 * <a for="built-in values" lt=subgroup_size>`'subgroup_size'`</a>
+* <a for="built-in values" lt=primitive_index>`'primitive_index'`</a>
```

Comment:
gotcha: primitive_index was missing before.

Reply:
Yes, the annoying maintenance burden of maintaining the list in multiple places :(

Reply:
cc: @dj2 who is always asking for redundancies... :-)